src/NuGet.Clients/VisualStudio/RestoreManager/IVsAssetIncludeFlags.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Asset include type flags
```

Comment:
Agree. Eventually I will. Just want to share interfaces as quickly as possible to collect some feedback.